ui/src/app/components/resource-list.tsx

```diff
@@ -221,14 +221,15 @@ export const ResourceList = fp.flow(withCdrVersions())((props: Props) => {
 
   return (
     <React.Fragment>
-      <div data-test-id='resources-table'>
+      <div data-test-id='resources-table' style={{ flex: 1 }}>
         {tableData?.length > 0 && (
           <DataTable
+            filterDisplay='row'
             data-test-id='resource-list'
             value={tableData}
-            scrollable={true}
             sortMode='multiple'
             paginator
+            breakpoint='0px'
```

Comment:
The new version of the datatable component has two modes: stack and scrolled. It defaults to stacked, but setting the breakpoint to 0px prevents the alternate view. I am up for having a more detailed conversation, if people are interested.